core/math/convex_hull.cpp

```diff
@@ -204,7 +204,7 @@ class ConvexHullInternal {
 		static Int128 mul(uint64_t a, uint64_t b);
 
 		Int128 operator-() const {
-			return Int128((uint64_t) - (int64_t)low, ~high + (low == 0));
+			return Int128((uint64_t)-(int64_t)low, ~high + (low == 0));
```

Comment:
```suggestion
			return Int128(uint64_t(-int64_t(low)), ~high + (low == 0));
```
If it works, this one is confusing to read in either situation

Reply:
That seems to work. I agree it's a bit better yeah.